src/FsAutoComplete/CodeFixes/AddMissingEqualsToTypeDefinition.fs

```diff
@@ -11,7 +11,7 @@ let title = "Add missing '=' to type definition"
 
 /// a codefix that adds in missing '=' characters in type declarations
 let fix (getFileLines: GetFileLines) =
-  Run.ifDiagnosticByCode (Set.ofList [ "3360" ]) (fun diagnostic codeActionParams ->
+  Run.ifDiagnosticByCode (Set.ofList [ "10" ]) (fun diagnostic codeActionParams ->
```

Comment:
Context: https://github.com/dotnet/fsharp/pull/15585#issuecomment-1633130376
`3360` doesn't seem to be a related thing anymore here:
https://github.com/dotnet/fsharp/blob/109a84d633917aa6e331187fc63eb509ef1a6dc1/src/Compiler/FSComp.txt#L1596